Assignment2/Assignment2_QNo2/SolAssignmentNo2_QNo2.rb

```diff
@@ -1,45 +1,55 @@
 # => 2. Write a class (with constructor) to define instance
 # => methods for getting area & perimeter of a rectangle.
 
-# => Buisness logic class
-class Rectangle
-  def initialize(length, breadth)
-    @length = length
-    @breadth = breadth
+# => Business logic class
+class RectangleSkeleton
+
+  attr_accessor :len, :br
```

Comment:
@HrisikeshSinghaChowdhury why extra space! There should not be any space between block start and first statement